frontend/src/features/contacts/contact/edit/Organization/UpdateOrganizationForm.tsx

```diff
@@ -61,7 +61,22 @@ export const UpdateOrganizationForm: React.FC<{ id: number }> = ({ id }) => {
   ]);
 
   const initialValues = !!formOrganization
-    ? { ...defaultCreateOrganization, ...formOrganization }
+    ? {
+        ...defaultCreateOrganization,
```

Comment:
...overwrites the subobjects, meaning that default values weren't being set in form for undefined values in the mailing/property/billing address coming from the api.